### PR TITLE
Cleanup Radio Programming Article

### PR DESCRIPTION
**Pull request by Daltz333**
- Fix punctuation
- Use guilabels
- Update images to Windows 10 for applicable parts
- Fix formatting

### CODE REVIEW

File: source/docs/zero-to-robot/step-2/radio-programming.rst [hunk 1]
```diff
@@ -10,12 +10,12 @@ Before you begin using the software:
 
 .. warning:: The OM5P-AN and AC use the same power plug as the D-Link DAP1522, however they are 12V radios. Wire the radio to the 12V 2A terminals on the VRM (center-pin positive).
 
-Pre-Requisites
---------------
+Prerequisites
+-------------
 
-The FRC Radio Configuration Utility requires the Java Runtime Engine (JRE). If you do not have Java installed, you can download the JRE from here: https://www.java.com/en/download/
+.. important:: Please note that the radio configuration utility requires Java 8, not Java 11 that is included with the WPILib installation. This can be installed alongside the WPILib Java by downloading and running the following installer with administrator privelages. Download `here <https://adoptopenjdk.net/?variant=openjdk8&jvmVariant=hotspot>`__.
```
Comment (prateekma) [inline source/docs/zero-to-robot/step-2/radio-programming.rst:16]: Check spelling of "privileges".

File: source/docs/zero-to-robot/step-2/radio-programming.rst [hunk 2]
```diff
@@ -84,14 +85,14 @@ Launch the software
 
 Use the Start menu or desktop shortcut to launch the program.
 
-Note: If you need to locate the program it is installed to C:\Program Files (x86)\FRC Radio Configuration Utility. For 32-bit machines the path is C:\Program Files\FRC Radio Configuration Utility\
+.. note:: If you need to locate the program, it is installed to ``C:\Program Files\FRC Radio Configuration Utility``. For 32-bit machines the path is ``C:\Program Files (x86)\FRC Radio Configuration Utility``
```
Comment (AustinShalit) [inline source/docs/zero-to-robot/step-2/radio-programming.rst:88]: It is located in `(x86)` regardless of how many bits the machines has.

File: source/docs/zero-to-robot/step-2/radio-programming.rst [hunk 3]
```diff
@@ -100,39 +101,39 @@ Select the network interface
...
 
-1. If no ethernet interfaces are listed, click "Refresh" to re-scan for available interfaces
-2. Select the interface you want to use from the drop-down list
-3. Click "OK"
+1. If no ethernet interfaces are listed, click :guilabel:`Refresh` to re-scan for available interfaces.
+2. Select the interface you want to use from the drop-down list.
+3. Click :guilabel:`OK`.
 
-Open Mesh Firmware Note
+Open-Mesh Firmware Note
```
Comment (AustinShalit) [inline source/docs/zero-to-robot/step-2/radio-programming.rst:108]: `Open Mesh` is correct

File: source/docs/zero-to-robot/step-2/radio-programming.rst [hunk 4]
```diff
@@ -100,39 +101,39 @@ Select the network interface
...
 For the FRC Radio Configuration Utility to program the OM5P-AN and OM5P-AC radio, the radio must be running an FRC specific build of the OpenWRT firmware. OM5P-AC radios in the 2019 KoP should not need an update.
 
 If you do not need to update or re-load the firmware, skip the next step.
 
-.. warning:: Note: Radios used in 2019 **do not** need to be updated before configuring, the 2020 tool uses the same 2019 firmware.
+.. warning:: Radios used in 2019 **do not** need to be updated before configuring, the 2020 tool uses the same 2019 firmware.
 
-Loading FRC Firmware to OpenMesh radio
---------------------------------------
+Loading FRC Firmware to Open-Mesh Radio
```
Comment (AustinShalit) [inline source/docs/zero-to-robot/step-2/radio-programming.rst:117]:
```suggestion
Loading FRC Firmware to Open Mesh Radio
```

File: source/docs/zero-to-robot/step-2/radio-programming.rst [hunk 5]
```diff
@@ -100,39 +101,39 @@ Select the network interface
...
+---------------------------------------
 
 .. image:: images/radio-programming/openmesh-firmware.png
 
 If you need to load the FRC firmware (or reset the radio), you can do so using the FRC Radio Configuration Utility.
 
 1. Follow the instructions above to install the software, launch the program and select the Ethernet interface.
-2. Make sure the OpenMesh radio is selected in the Radio dropdown.
+2. Make sure the Open-Mesh radio is selected in the Radio dropdown.
```
Comment (AustinShalit) [inline source/docs/zero-to-robot/step-2/radio-programming.rst:125]:
```suggestion
2. Make sure the Open Mesh radio is selected in the Radio dropdown.
```

File: source/docs/zero-to-robot/step-2/radio-programming.rst [hunk 6]
```diff
@@ -146,16 +147,16 @@ Select Options
...
-3. BW Limit: If this box is checked, the radio enforces a 4 Mbps bandwidth limit like it does when programmed at events. Note that in AP mode, this is a total limit, not per client, so streaming video to multiple clients simultaneously may cause undesired behavior.
+1. **Robot Name:** This is a string that gets appended to the SSID used by the radio. This allows you to have multiple networks with the same team number and still be able to distinguish them.
+2. **Firewall:** If this box is checked, the radio firewall will be configured to attempt to mimic the port blocking behavior of the firewall present on the FRC field. For a list of open ports, please see the FRC Game Manual.
+3. **BW Limit:** If this box is checked, the radio enforces a 4 Mbps bandwidth limit like it does when programmed at events. Note that in AP mode, this is a total limit, not per client, so streaming video to multiple clients simultaneously may cause undesired behavior.
 
-.. note:: Firewall and BW Limit only apply to the OpenMesh radios. These options have no effect on D-Link radios.
+.. note:: Firewall and BW Limit only apply to the Open-Mesh radios. These options have no effect on D-Link radios.
```
Comment (AustinShalit) [inline source/docs/zero-to-robot/step-2/radio-programming.rst:154]:
```suggestion
.. note:: Firewall and BW Limit only apply to the Open Mesh radios. These options have no effect on D-Link radios.
```

File: source/docs/zero-to-robot/step-2/radio-programming.rst [hunk 7]
```diff
@@ -146,16 +147,16 @@ Select Options
...
 
-.. warning:: The "Firewall" option configures the radio to emulate the field firewall. This means that you will not be able to deploy code wirelessly with this option enabled.
+.. warning:: The "Firewall" option configures the radio to emulate the field firewall. This means that you will not be able to deploy code wirelessly with this option enabled. This is useful for simulating blocked ports that may exist at competitions.
```
Comment (AustinShalit) [inline source/docs/zero-to-robot/step-2/radio-programming.rst:156]:
```suggestion
.. warning:: The "Firewall" option configures the radio to emulate the field firewall. This means that you will not be able to deploy code wirelessly with this option enabled. This is useful for simulating blocked ports that exist at competitions.
```

File: source/docs/zero-to-robot/step-2/radio-programming.rst [hunk 8]
```diff
@@ -84,14 +85,14 @@ Launch the software
...
-Note: If you need to locate the program it is installed to C:\Program Files (x86)\FRC Radio Configuration Utility. For 32-bit machines the path is C:\Program Files\FRC Radio Configuration Utility\
+.. note:: If you need to locate the program, it is installed to ``C:\Program Files\FRC Radio Configuration Utility``. For 32-bit machines the path is ``C:\Program Files (x86)\FRC Radio Configuration Utility``
 
 Allow the program to make changes, if prompted
 ----------------------------------------------
 
 .. image:: images/radio-programming/allow-changes.png
 
-If the your computer is running Windows Vista or Windows 7, a prompt may appear about allowing the configuration utility to make changes to the computer. Click "Yes" if the prompt appears.
+If the your computer is running Windows Vista or Windows 7, a prompt may appear about allowing the configuration utility to make changes to the computer. Click :guilabel:`Yes` if the prompt appears.
```
Comment (AustinShalit) [inline source/docs/zero-to-robot/step-2/radio-programming.rst:95]:
```suggestion
If the your computer is running Windows 7, a prompt may appear about allowing the configuration utility to make changes to the computer. Click :guilabel:`Yes` if the prompt appears.
```

**Review by AustinShalit — CHANGES_REQUESTED** [review]

**Review by AustinShalit — APPROVED** [review]

**Pull request merged**